config/locales/en.yml

```
@@ -1206,7 +1211,9 @@ en:
         residence_history: Provide details and dates of all previous addresses for the last 5 years
       steps_respondent_contact_details_form:
         <<: *PERSONAL_CONTACT_FIELDS
-        residence_history: Please provide details of all previous addresses for the last 5 years below (if known, including the dates and starting with the most recent)
+      steps_respondent_address_details_form:
+        <<: *PERSONAL_ADDRESS_FIELDS
+        residence_history: Provide details and dates of all previous addresses for the last 5 years
```

Better to leave this as originally:

```suggestion
        residence_history: Please provide details of all previous addresses for the last 5 years below (if known, including the dates and starting with the most recent)
```